### DISCUSSION TITLE
How to disable all autoResize features?

### DISCUSSION THREAD

**Discussion by KaitamakiKalle** [ghiscoding/slickgrid-universal#1540] [Q&A] [answered]
Hi, how can I disable autoResizing features completely? `autoResize` seems to have problems when used with `flexbox`.  Normal slickgrid (6pac repo) seems to operate correctly in same scenario. I have tried to disable `autoResize = false` in gridOptions and it seems like it doesn't disable all automatic resizing features. I also found `global-grid-options.js` file and tried disabling it there but doesn't seem to work.

**Comment by ghiscoding** [comment, marked as answer]
`autoResize` is not a boolean but rather an object accepting other options, you'll want to use `enableAutoResize` instead. Also if you set a `gridHeight` and `gridWidth` (which I assume you will need without the auto-resize) then it disable the resize (e.g. live demo Example 1). TypeScript intellisense in VSCode should show the correct type.

You might want to try `autoResize: { resizeDetection: 'container' }` option that was added by another user, you may have better result. 

> I also found `global-grid-options.js` file and tried disabling it there but doesn't seem to work.

you should never change a file from the library itself, this will be overwritten every time you run npm install
